src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/NotificationsService.cs

```diff
@@ -30,17 +28,17 @@ public async Task NotifySubscribersIfApplicable(params Guid[] releaseIds)
         {
             var releasesToNotify = _context.Releases
                 .Include(r => r.ReleaseStatuses)
+                .Include(r => r.Updates) // used by BuildPublicationNotificationMessage
+                .Include(r => r.Publication) // used by BuildPublicationNotificationMessage
                 .Where(r => releaseIds.Contains(r.Id))
+                .Distinct()
```

Comment:
I see the `Distinct` has probably been moved from below in the method that use to be `GetPublications` but I'm not sure what it's doing here. I think each row is going to be distinct by release id and update id and release status id already so there shouldn't be any duplicates.

Reply:
I've removed the `Distinct`